Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/web/QuizList.jsp

```diff
@@ -0,0 +1,93 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%--
+  Created by IntelliJ IDEA.
+  User: Username
+  Date: 21.04.2018
+  Time: 20:08
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Quiz List</title>
+    <link href="styles.css" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+</head>
+<body>
+<div class="container">
+
+    <div class="mainblock">
+
+
+        <!--New quiz -->
+        <p>Create new quiz!</p>
+
+        <form class="form-style-4" name="formQuiz" method="post" action="newquiz">
+            <table>
+                <tr>
+                    <td>Test title</td>
+                    <td><input type="text" name="testTitle"></td>
+                </tr>
+                <tr>
+                    <td>Objective</td>
+                    <td><input type="text" name="objective"></td>
+                </tr>
+            </table>
+
+
+            <input hidden name="formCreateQuiz" value="true">
+            <input type="submit" name="addTest" value="Add quiz">
+
+        </form>
+        <!--quiz list -->
+
+
+        <table class="table_quiz">
+            <caption/>
+            <tbody>
+            <tr>
+                <th>ID</th>
+                <th>Title</th>
+                <th>Objective</th>
+                <th>Creator</th>
+                <th/>
+            </tr>
+            <c:forEach items="${list}" var="test">
+                <tr>
+                    <input type="hidden" name="table_id" value="${test.getId()}"/>
+                    <td> ${test.getId()}</td>
+                    <td> ${test.getName()}</td>
+                    <td> ${test.getObjective()} </td>
+                    <td> ${test.getCreator()} </td>
+                    <td class="deleteRow">Delete</td>
+
+                    <!--<td><a href="delQuiz?id=${test.getId()}" content=" ">Delete</a></td>-->
+                </tr>
+            </c:forEach>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+<script type="text/javascript">
+    $(document).ready(function () {
+        $(".deleteRow").click(function (evt) {
+            var selectId = $(this).closest("tr").find("[name='table_id']").val();
+
+            $.ajax({
+                type: "GET",
```

Comment:
If you send request via ajax you can choose write method for it. Why 'get' for deletion?

Reply:
Do not quite understand. Before that, I had a get request that worked with deleting the selected row from the collection. Now the asynchronous method works with this.
 What's wrong? Or what other methods do you mean?

Reply:
With other methods I mean DELETE Http method. Which is designed to do deletion